ui/app/src/components/SingleFileUpload/SingleFileUpload.tsx

```diff
@@ -61,7 +61,7 @@ const messages = defineMessages({
   },
   policyError: {
     id: 'fileUpload.policyError',
-    defaultMessage: 'The upload file name goes against project policies.'
+    defaultMessage: 'File "{fileName}" doesn\'t comply with project policies and can\'t be uploaded'
```

Comment:
Updated this message to match the one in XB, since the policy is not only for fileName